### PR TITLE
 fix(docs): remove empty table header for project maintainers section 

### PR DESCRIPTION
**Pull request by kerolloz**
[This](https://github.com/strongloop/loopback-next/pull/3181) pull request was automatically closed and I couldn't reopen it.
>in the project maintainers section
I just turned markdown to HTML. because GitHub flavored markdown doesn't support tables without headers.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] New tests added or existing tests modified to cover all changes
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,14 +87,42 @@ You can join the team by posting a comment to
...
-| :----------------------------------------------: | :----------------------------------------------: | :--------------------------------------------------------: |
-|                  Biniam Admikew                  |                    Diana Lau                     |                         Janny Hou                          |
-|    [![b-admike]](http://github.com/b-admike)     |      [![dhmlau]](http://github.com/dhmlau)       |         [![jannyhou]](http://github.com/jannyHou)          |
-|                    Hage Yaapa                    |                 Nora Abdelgadir                  |                       Mario Estrada                        |
-| [![hacksparrow]](https://github.com/hacksparrow) | [![nabdelgadir]](https://github.com/nabdelgadir) | [![marioestradarosa]](https://github.com/marioestradarosa) |
-|                   Hugo Da Roit                   |                 Dominique Emond                  |                         Agnes Lin                          |
-|        [![yaty]](https://github.com/yaty)        |     [![emonddr]](https://github.com/emonddr)     |         [![agnes512]](https://github.com/agnes512)         |
+<table>
```
Comment (marioestradarosa) [inline README.md]: Not good idea to place HTML here.  Even the documentation is MarkDown based. I believe the GitHub flavored parser don't support table without headers anyway.

Comment (bajtos) [reply README.md]: @marioestradarosa 

>  I believe the GitHub flavored parser don't support table without headers anyway.

AFAICT, GitHub's markdown parser handles HTML very well. Here is a screenshot from https://github.com/strongloop/loopback-next/blob/796e87898d929dfced7a1de215b854d1a01f3b88/README.md:

Screen Shot 2019-06-24 at 09 57 04

> Not good idea to place HTML here. Even the documentation is MarkDown based.

Could you please go into more details why you think it's not a good idea to use HTML?

Are you concerned about inconsistency - some places using Markdown tables and other places using raw HTML? Anything else?

Personally, I am fine to use raw HTML in places where it make sense, as an exception to the general rule of preferring Markdown tables everywhere else.

@strongloop/loopback-maintainers thoughts?

Comment (dhmlau) [reply README.md]: I think my question in mind is it seems to be rendering fine (at least to me) currently in https://github.com/strongloop/loopback-next#team, why do we need to change it?  Or maybe I'm not experiencing the problem @kerolloz is seeing.

Comment (nabdelgadir) [reply README.md]: Screen Shot 2019-06-24 at 11 11 41 AM
I see it on my end too.

**Comment by bajtos** [comment]
Thank you @kerolloz for the pull request again. While the current Markdown-based table has few shortcomings (extra header, difficult to edit), I feel the HTML version is ever worse because of the complexity involved around building a table cell with an image link:

```html
<td>
<a href="https://github.com/marioestradarosa">
<img src="https://avatars2.githubusercontent.com/u/4633823?v=3&amp;s=60" alt="marioestradarosa" style="max-width:100%;">
</a></td>
```

Let's keep the current Markdown-based version for now.

**Pull request closed without merging**